cadence/gateway/cadence/client.py

```diff
@@ -0,0 +1,33 @@
+from typing import List, Tuple
+import grpc
+import uber.cadence.api.v1.service_domain_pb2 as service_domain_pb2
+import uber.cadence.api.v1.service_domain_pb2_grpc as service_domain_pb2_grpc
+
+
+class WorkflowClient(object):
+    """
+    Client for gRPC functionality
+    """
+    host = 'localhost'
+    server_port = 7833
```

Comment:
@longquanzheng do you think the final customer should have the chance to provide this parameters to the cadence client?

Reply:
Yes I think they should. The ports are configurable on server